docs/docs/user/references/editor-integration.md

```diff
@@ -13,10 +13,13 @@ KPOps provides JSON schemas that enable autocompletion and validation for some o
 [yaml-language-server](https://github.com/redhat-developer/yaml-language-server#clients){target=_blank} in your editor of choice. (requires LSP support)
 2. Configure the extension with the settings below.
 
-???+ note "`settings.json`"
+??? note "`settings.json`"
 
     ```json
       --8<--
       ./docs/resources/editor_integration/settings.json
       --8<--
     ```
+
+!!! tip "Advanced usage"
+    It is possible to generate schemas with the [`kpops schema`](../cli-commands/#kpops-schema) command. Useful when using custom components or when using a pre-release version of KPOps.
```

Comment:
in the future we should update this section with a small guide on how the custom schema generation can be integrated into a project, because this is a really cool feature to have 🙂

Reply:
Agreed, created an [issue](https://github.com/bakdata/kpops/issues/173) for it.